src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -23,7 +26,7 @@ export class RepresentationMetadata {
    *
    * `@ignored` tag is necessary for Components-Generator.js
    */
-  public constructor(identifier?: NamedNode | BlankNode | string, overrides?: Record<string, MetadataOverrideValue>);
+  public constructor(identifier?: MetadataIdentifier, overrides?: Record<string, MetadataOverrideValue>);
```

Comment:
I would have been ok with only allowing ResourceIdentifiers here even, but this also works.

Reply:
Ah, I doubted. In  `src`, it's only used by `src/util/ResourceUtil.ts`. But there was more in the tests, so I thought you might want to use it in the future.